src/easylink/rule.py

```diff
@@ -201,12 +206,16 @@ def _build_resources(self) -> str:
 
     def _build_shell_cmd(self) -> str:
         """Builds the shell command portion of the rule."""
-        # TODO: handle multiple wildcards
-        # output_paths = ",".join(self.output)
-        # wildcards_subdir = "/".join([f"{{wildcards.{wc}}}" for wc in self.wildcards])
-        # and then in shell cmd: export DUMMY_CONTAINER_OUTPUT_PATHS={output_paths}/{wildcards_subdir}
+        # TODO [MIC-5877]: handle multiple wildcards, e.g.
+        #   output_paths = ",".join(self.output)
+        #   wildcards_subdir = "/".join([f"{{wildcards.{wc}}}" for wc in self.wildcards])
+        #   and then in shell cmd: export DUMMY_CONTAINER_OUTPUT_PATHS={output_paths}/{wildcards_subdir}
+        if len(self.output) > 1:
+            raise NotImplementedError(
+                "FIXME [MIC-5883] Multiple output files not yet supported"
+            )
```

Comment:
Doesn't this need to be inside the `if self.is_embarrassingly_parallel`? If so, can we add a test that would fail due to this?